docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/movement/strided-slice-1.rst

```diff
@@ -14,6 +14,69 @@ StridedSlice
+
+.. note:: Value Out-of-Bounds (Silent Clamping)
+
+   If a value in begin or end is out of bounds for the corresponding dimension, it is silently clamped. In other words:
+
+   * If :math:`begin[i] >= size(dim)`, then :math:`begin[i] = size(dim)`. If :math:`begin[i] < 0` (after Negative Values Adjusting), then :math:`begin[i] = 0`.
+   * If :math:`end[i] >= size(dim)`, then :math:`end[i] = size(dim)`. If :math:`end[i] < 0` (after Negative Values Adjusting), then :math:`end[i] = 0`.
+
+   If slicing in reverse, the clamping behavior changes to the following:
+
+   * If :math:`begin[i] >= size(dim)`, then :math:`begin[i] = size(dim) - 1`. If :math:`begin[i] < 0` (after Negative Values Adjusting), then :math:`begin[i] = 0`.
+   * If :math:`end[i] >= size(dim)`, then :math:`end[i] = size(dim)`. If :math:`end[i] < 0` (after Negative Values Adjusting), then :math:`end[i] = -1`.
+
+The operation accepts multiple bitmasks in the form of integer arrays to modify the above behavior. **If the length of the bitmask is less than the length of the corresponding input, it is assumed that the bitmask is extended with zeros.**
+
+During the i-th slicing step:
+
+* If the :math:`begin\_mask[i]` is set to one, the value of :math:`begin[i]` is set to 0 (size(dim) - 1 if slicing in reverse).
+* If the :math:`end\_mask[i]` is set to one, the value of :math:`end[i]` is set to size(dim) (0 if slicing in reverse - note that this does not allow slicing inclusively with the first value).
+* If the :math:`new\_axis\_mask[i]` is set to one, the values of :math:`begin[i]`, :math:`end[i]`, and stride[i] ARE IGNORED, and a new dimension with size 1 appears in the output. No slicing occurs at this step.
+* If the :math:`shrink\_axis\_mask[i]` is set to one, the values of :math:`begin[i]` **MUST EQUAL** :math:`end[i]` (Note that this would normally result in a size 1 dimension), and the **stride[i]** value IS IGNORED. The corresponding dimension is removed, with only a single element from that dimension remaining.
+* If the :math:`ellipsis\_mask[i]` is set to one, the :math:`begin[i], end[i], and stride[i]` **ARE IGNORED**, and a number of dimensions are skipped. The exact number of dimensions skipped in the original input is :math:`size(dim) - (M - sum(new\_axis\_mask) - 1)`. The corresponding dimension is treated as an ellipsis ('...'), or in other words, it is treated as multiple, sequential, and unaffected by slicing dimensions, that match the rest of the slicing operation. This allows for a concise and flexible way to perform slicing operations, effectively condensing the slicing parameters for dimensions marked with ellipsis into a single slice notation. For example, given a 10D input, and tasked to select the first element from the 1st and last dimension, normally one would have to write :math:`[0, :, :, :, :, :, :, :, :, :, 0]`, but with ellipsis, it is only necessary to write :math:`[0, ..., 0]`.
```

Comment:
What will be if there are `begin= [0, 1, 10, 10]`, `begin_mask = [0, 0, 1, 1]` and `ellipsis_mask =[0, 1]` (Other masks are empty)? 
Could you clarify please how to cover this case? What we should do the latest dimensions?

Reply:
I have added this as an example, with matching operation equivalent in Python for clarity